docs/tutorials/excel-tutorial.md

```diff
@@ -19,29 +19,29 @@ In this tutorial, you'll create an Excel task pane add-in that:
 > * Protects a worksheet
 > * Opens a dialog
 
-## Prerequisites
-
-To use this tutorial, you need to have the following installed. 
-
-- Excel 2016, version 1711 (Build 8730.1000 Click-to-Run) or later. You might need to be an Office Insider to get this version. For more information, see [Be an Office Insider](https://products.office.com/office-insider?tab=tab-1).
+> [!TIP]
+> If you've already completed the [Build an Excel task pane add-in](../quickstarts/excel-quickstart-jquery.md) quick start, and want to use that project as a starting point for this tutorial, go directly to the [Create a table](#create-a-table) section to start this tutorial.
 
-- [Node](https://nodejs.org/en/) 
-
-- [Git Bash](https://git-scm.com/downloads) (or another Git client)
+## Prerequisites
 
-- You need to have an Internet connection to test the add-in in this tutorial.
+[!include[Yeoman generator prerequisites](../includes/quickstart-yo-prerequisites.md)]
 
 ## Create your add-in project
 
-Complete the following steps to create the Excel add-in project that you'll use as the basis for this tutorial.
+Use the Yeoman generator to create an Excel add-in project. Run the following command and then answer the prompts as follows:
 
-1. Clone the GitHub repository [Excel add-in tutorial](https://github.com/OfficeDev/Excel-Add-in-Tutorial).
+```command&nbsp;line
+yo office
+```
 
-2. Open a Git bash window, or Node.JS-enabled system prompt, and navigate to the **Start** folder of the project.
+- **Choose a project type:** `Office Add-in Task Pane project`
```

Comment:
Actually, there are two questions about telemetry right after yo office is run. Both asking for permission. The first is from the makers of yeoman and the second is from Microsoft.
![image](https://user-images.githubusercontent.com/9088056/64999330-4a074900-d89c-11e9-973a-09416f168b2f.png)

Reply:
These questions about telemetry only appear the first time you run version 1.4.0 or greater of Yo Office -- but it's probably worth mentioning nonetheless. I'll add a note to each tutorial, and to other places in the docs that contain instructions for using Yo Office as well.